discovery.md

```diff
@@ -110,6 +111,76 @@ be considered an event subscriber and a target user of this API.
 This API is specified as a REST API with well defined entities and relationships
 between those entities.
 
+To help explain the data model, the following non-normative pseudo yaml shows
+its basic conceptual structure:
+
+( `*` means zero or more, `+` means one or more, `?` means optional)
+
+```
+- sources: *
+  - sourceclass: [unique identifier for producer of the specified event types]
```

Comment:
I really don't like 'sourceclass' - I think the word class makes it sound general/abstract

How about 'sourcesystem' ?

Reply:
Imho it's not a system. For many of us it will be something like a product or service. _System_ actually suggests that it is the same as the producer. To me it really is a lot like a class. I think of the source as an instance of it. A sourceclass could be repository.github.com, and a specific source would be of form github.com/{org}/{repo}. 
Perhaps we still don't have the same understanding of the semantic of this new concept. I see at least three different possible meanings:

1. Related to physical entity: system, producer, host
2. Describe common properties and meaning: class, category, type, prototype 
3. No semantics, just multiple sources bundled together: group, family

Reply:
what about "service"? 

I want to subscribe to the "object store service".
github, gitlab, ... are services, right? And each has a set of "sources" about which you can ask for events.
Maybe?

Reply:
I could live with "service". I suppose `source` will be many different things. @clemensv often comes up with IoT examples. Sometimes you will have one source service/class/system per source. Sometimes it will be many. As we give this thing an id, I wonder what exactly it identifies. 
Btw. what about `sourcespec`? 😄